transform/composite_indexer.go

```diff
@@ -10,13 +10,15 @@ import (
 // CompositeIndexer delegates to subindexers to transform resources to solr Documents
 type CompositeIndexer struct {
 	publicationIndexer Indexer
+	personsIndexer     Indexer
```

Comment:
Should this be `personIndexer` for plurality consistency w/ `publicationIndexer`?